scripts/proto.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/python
```

Comment:
We should follow the pattern used in the other scripts --- see download_data.py as good example.

- Remove shebang, and chmod a-x
- Add docstring comments describing script. What is it for? How should it be used?
- Use usual import structure, including futures.